Sources/GetStartedApp/CloudantDatabaseManager.swift

```diff
@@ -23,11 +23,23 @@ import SwiftyJSON
 
 class CloudantDatabaseManager: DatabaseManager {
 
-  private var db: Database? = nil
+  private var cloudant: Database?
   private let dbClient: CouchDBClient
   private let semaphore = DispatchSemaphore(value: 1)
   private let dbName: String
 
+  struct User: Document {
```

Comment:
There is already a `User` class in `Controller.swift`. I think It would make more sense to use this class (Or make it a struct) and make it conform to document instead of creating a new struct.